### DISCUSSION TITLE
Strange network behavair..

### DISCUSSION THREAD

**Discussion by Dylantje** [Koenkk/zigbee2mqtt#8357] [General]
Hi like the head sasy..
I do have:
![ScreenShot141](https://user-images.githubusercontent.com/5214969/129343301-361f90c3-1f21-4676-8594-04bb0fe5d418.png)
In domoticz:
![ScreenShot142](https://user-images.githubusercontent.com/5214969/129343332-79e36f6d-cd82-4abd-877c-3f3ca0d0c628.png)

Qustion

Why are there so much slave devices not in connetion with the routers?
When i learn them, they look fine, the are working fine etc. Only i think it is better for the mesh that they are in to the network.

So how?
Is there a command? of what can i do for a better mesh network:-)

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/information/FAQ.html#why-are-some-links-missing-from-my-networkmap

**Comment by Dylantje** [comment]
THANKS Mate!!!!
Sorry for my noob/dombo questions!!!

Love your great work!
